### PR TITLE
[ADD] {de,re}activate_workflow_transitions api

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: openerp/openupgrade/openupgrade.py [hunk 1]
```diff
@@ -547,6 +549,65 @@ def message(cr, module, table, column,
...
+
+    Returns a dictionary to be used on reactivate_workflow_transitions
+
+    :param model: the model for which workflow transitions should be
+    deactivated
+    :param transitions: a list of ('module', 'name') xmlid tuples of
+    transitions to be deactivated. Don't pass this if there's no specific
+    reason to do so, the default is to deactivate all transitions
+
+    .. versionadded:: 7.0
```
Comment (legalsylvain) [inline openerp/openupgrade/openupgrade.py:566]: reading the whole openupgrade.py file, I guess that versionadded has to indicate the target version and not the original. (Must be 8.0)

File: openerp/openupgrade/openupgrade.py [hunk 2]
```diff
@@ -547,6 +549,65 @@ def message(cr, module, table, column,
...
+
+def reactivate_workflow_transitions(cr, transition_conditions):
+    """
+    Reactivate workflow transition previously deactivated by
+    deactivate_workflow_transitions.
+
+    :param transition_conditions: a dictionary returned by
+    deactivate_workflow_transitions
+
+    .. versionadded:: 7.0
```
Comment (legalsylvain) [inline openerp/openupgrade/openupgrade.py:603]: the same here.

File: openerp/openupgrade/openupgrade.py [hunk 3]
```diff
@@ -547,6 +549,65 @@ def message(cr, module, table, column,
...
+    deactivated
+    :param transitions: a list of ('module', 'name') xmlid tuples of
+    transitions to be deactivated. Don't pass this if there's no specific
+    reason to do so, the default is to deactivate all transitions
+
+    .. versionadded:: 7.0
+    """
+    transition_ids = []
+    if transitions:
+        for module, name in disable_transitions:
```
Comment (legalsylvain) [inline openerp/openupgrade/openupgrade.py]: Where is 'disable_transitions' defined ? did you meant 'transitions' ?

**Comment by legalsylvain** [comment]
Duplicated PR ? (#163)

**Comment by hbrunn** [comment]
no. #163 is for 7.0, this is for 8.0. That's where the versionadded part comes from. You comment about disable_transitions was valid for both branches though

**Comment by legalsylvain** [comment]
How sorry, I did not noticed ...
Copy / past fixed, so LGTM :+1:. Thanks

**Comment by pedrobaeza** [comment]
Why the conflict?

**Comment by hbrunn** [comment]
good question. I cherry picked the commit from the 7.0 branch, resolved the conflict that was created and pushed that. Weird. Seems I have to rebase this

**Comment by StefanRijnhart** [comment]
:1+: after a rebase (same as #163)

**Comment by hbrunn** [comment]
superseded by #173

**Pull request closed without merging**